### DISCUSSION TITLE
Remove me please

### DISCUSSION THREAD

**Discussion by imdevarsh** [JaydenDev/freecat#51] [Q&A]
Okay, sorry guys, I made a mistake, but I don't want to have push access due to my IP address... can you please remove me? Thanks.

**Comment by hello-smile6** [comment]
> Okay, sorry guys, I made a mistake, but I don't want to have push access due to my IP address... can you please remove me? Thanks.

Why? They already have your IP address, why not just keep it as-is?

**Reply by imdevarsh** [reply]
Okay, fine.

**Comment by imdevarsh** [comment]
I don't want to work on Freecat now...

**Reply by hello-smile6** [reply]
Why?
